packages/server/test/integration/cypress_spec.js

```diff
@@ -1110,7 +1111,7 @@ describe('lib/cypress', () => {
       })
 
       // TODO: handle PORT_IN_USE short integration test
-      it('logs error and exits when port is in use', function () {
+      xit('logs error and exits when port is in use', function () {
```

Comment:
Did you mean to keep this as `xit`?

Reply:
I fixed it up: https://github.com/cypress-io/cypress/pull/17200/commits/dbd14a36941cb317a8ebaaa82cc7451c13cea8ce

Not ideal (neither was original implementation). I also was unable to actually trigger this error in practice - I have a feeling this is testing a scenario which cannot happen in production.